extension/src/experiments/columns/collect/timestamp.ts

```diff
@@ -4,8 +4,8 @@ import { ColumnType } from '../../webview/contract'
 export const timestampColumn = {
   hasChildren: false,
   label: 'Timestamp',
-  parentPath: 'timestamp',
-  path: 'timestamp:timestamp',
+  parentPath: ColumnType.TIMESTAMP,
```

Comment:
I would see if it is possible to change this to an optional property and drop it from here.